File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/PackagesUpdatePackageRemediateRequestItem.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class PackagesUpdatePackageRemediateRequestItem(
    @JsonProperty("packageLabel")
    val packageLabel: String? = null,
    @JsonProperty("remediationDate")
    val remediationDate: String? = null,
    @JsonProperty("remediationMethodName")
    val remediationMethodName: String? = null,
    @JsonProperty("remediationSteps")
    val remediationSteps: String? = null
)

File: sdks/thunkmetrc-go/wrapper/models/packages_update_package_remediate_request_item.go
package models

type PackagesUpdatePackageRemediateRequestItem struct {
    PackageLabel string `json:"PackageLabel,omitempty"`
    RemediationDate string `json:"RemediationDate,omitempty"`
    RemediationMethodName string `json:"RemediationMethodName,omitempty"`
    RemediationSteps string `json:"RemediationSteps,omitempty"`
}
